Cargo.toml

```diff
@@ -47,6 +47,7 @@ router = "0.5"
 handlebars-iron = "0.22"
 params = "0.8"
 staticfile = { version = "0.4", features = [ "cache" ] }
+rand = "0.7.3"
```

Comment:
Please make this a dev-dependency. I know it doesn't make a difference in practice since it's a transitive dependency but it communicates intent more clearly.